test/spec/modules/roxotAnalyticsAdapter_spec.js

```diff
@@ -1,10 +1,27 @@
 import roxotAnalytic from 'modules/roxotAnalyticsAdapter';
+import { expect } from 'chai';
 let events = require('src/events');
 let adaptermanager = require('src/adaptermanager');
 let constants = require('src/constants.json');
 
 describe('Roxot Prebid Analytic', function () {
   describe('enableAnalytics', function () {
+    it('should enable roxot analytics', () => {
+      sinon.spy(roxotAnalytic, 'enableAnalytics');
+      adaptermanager.registerAnalyticsAdapter({
+        code: 'roxot',
+        adapter: roxotAnalytic
+      });
+
+      adaptermanager.enableAnalytics({
+        provider: 'roxot',
+        options: {
+          publisherIds: ['test_roxot_prebid_analytid_publisher_id']
+        }
+      });
+      sinon.assert.calledOnce(roxotAnalytic.enableAnalytics);
+      roxotAnalytic.enableAnalytics.restore();
```

Comment:
sinon `spies` & `restores` need to be in `beforeEach` and `afterEach` blocks.

Otherwise functions don't get restored in the event of a test failure, and then the failure cascades into other tests (since `sinon` errors if you try to spy on a spy).